app/controllers/volunteers_controller.rb

```diff
@@ -10,53 +12,58 @@ def index
     # profile page.
     # @repair = Repair.all
     @volunteer = Volunteer.all
+    authorize Volunteer
```

Comment:
```suggestion
    authorize @volunteer
```

Reply:
I think it is `authorize Volunteer` in this case.  `@volunteer` above should be `@volunteers` , but I don't think you can pass a collection of objects to a policy.  See https://github.com/varvet/pundit#policies

Reply:
Thanks Leila and John!

Reply:
I don't know very much about Pundit, but `authorize @volunteer` was used for all of the other methods in this controller, so I had assumed they should all be the same. 

Based on the link that you provided @jrmcgarvey, it looks like it should be `@volunteer`, though:
![image](https://user-images.githubusercontent.com/42787645/55769254-0dfacd00-5a4e-11e9-9bfa-40e3bc8ce64f.png)

Reply:
The following link has done `authorize @articles` for `index` action. https://code.tutsplus.com/tutorials/authorization-with-pundit--cms-28202

Reply:
Alright, I've spent some more time reading through the Pundit documentation, and now I think that you were right @jrmcgarvey. It does look like calling `authorize Volunteer` is the correct code for the `index` method. 

However, the Pundit `VolunteerPolicy` only has a method defined for the `#index` action, so calling `authorize` in all of the methods in the volunteer controller won't have any effect (or perhaps won't work at all).

Reply:
So, I'll leave as is.

Reply:
Oh, sorry @narayanrai97 -- looks like you responded right before I did last night, and I didn't see your message! The tutorial that you linked to certainly does show an example like `authorize @volunteers`, so perhaps that's just fine! I'm honestly not sure, so maybe test it out both ways?

Either way on that one, you'll still need to address the issue that you don't have any policies built for the other methods in the volunteer controller but you're still trying to call `authorize` in each of them.